cmd/cleanup.go

```diff
@@ -0,0 +1,45 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func CleanupCmd(fs afero.Fs) *cobra.Command {
+	cleanupCmd := &cobra.Command{
+		Use:  "cleanup",
+		Long: "Delete all fragments",
+		Args: func(cmd *cobra.Command, args []string) error {
+			return nil
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			path, err := cmd.Flags().GetString("path")
+			if err != nil {
+				return fmt.Errorf("error parsing flag 'path': %w", err)
+			}
+
+			err = fs.RemoveAll(path)
+			if err != nil {
+				return fmt.Errorf("error deleting fragments: %w", err)
+			}
+
+			err = fs.Mkdir(path, os.ModePerm)
+			if err != nil {
+				return fmt.Errorf("error creating fragments folder: %w", err)
+			}
+
+			return nil
+		},
+	}
+
+	cleanupCmd.Flags().String("path", "changelog/fragments", "The folder where fragments are stored.")
```

Comment:
Remove this flag, as is possible to override this value overriding [`fragment_path`](https://github.com/elastic/elastic-agent-changelog-tool/blob/f2e8d6613fc17aeba181e44f3e2e024b59a36d9b/internal/settings/settings.go#L56) configuration setting.